bin/merge_gene_attributes.sh

```diff
@@ -10,17 +10,14 @@ scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 IFS="
 "
 
-## check env variables
-[ ! -z ${ANNOTATION_PATH+x} ] || (echo "Env var ANNOTATION_PATH not defined." && exit 1)
-[ ! -z ${OUTPUT_PATH+x} ] || (echo "Env var OUTPUT_PATH not defined." && exit 1)
```

Comment:
Any reason why we are not checking for these anymore? or for the new env vars that are being used instead?

Reply:
`ANNOTATION_PATH` is not used directly by the script, it uses individual files from that directory as an input. So it makes more sense to check for it outside the script. 
`OUTPUT_PATH` was changed into `GENE_ATTRIBUTES_PATH` to reflect the meaning of the variables better. There is a check for that.